updated packages to latest versions

dhuang612 opened the pull request:
reviewed packages
updated popper.js to v1.15.0 - which removed the following error:
```
Error: Bootstrap dropdown require Popper.js (https://popper.js.org)
```
updated bootstrap css and js to version 4.3.1

checked out jquery which is only 1 version behind and didn't appear to be critically out of date
cache-polyfill appears to be up to date since it's just a piece of logic from this repo:
https://github.com/mathiasbynens/cache-polyfill

Please let me know if any changes need to be made

dhuang612 commented:
Visual studio code made the format changes on index.html
I disabled the setting and reformatted the document and re-submitted the PR

jancborchardt commented:
@dhuang612 nice! Only 2 last things:

- Could you use the minified version of Popper? E.g. the one you get from https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js
- Is it possible to also remove that last leftover change to the index.html from the pull request? It might create problems with other pull requests which work on index.html, even though there’s no actual change here. :)

dhuang612 commented:
@jancborchardt 
added in the extra space at the end of index.html
github is now showing no change for that file.

grabbed the min version of popper and replaced the version included in this PR
squashed extra commit.

Please let me know if anything else needs to be changed
